app/models/pipeline_run_stage.rb

```diff
@@ -262,7 +262,7 @@ def host_filtering_command
     batch_command = [install_pipeline(pipeline_run.pipeline_commit), upload_version(pipeline_run.pipeline_version_file), dag_commands].join("; ")
 
     # Dispatch job. Use the himem settings for host filtering.
-    aegea_batch_submit_command(batch_command, vcpus: Sample::DEFAULT_VCPUS_HIMEM, job_queue: Sample::DEFAULT_QUEUE_HIMEM, memory: Sample::HIMEM_IN_MB)
+    aegea_batch_submit_command(batch_command, vcpus: Sample::DEFAULT_VCPUS_HIMEM, job_queue: Sample::DEFAULT_QUEUE_HIMEM, memory: Sample::HIMEM_IN_MB, stage_name: "hostfilter", sample_id: sample.id)
```

Comment:
nit: could we change these to DAG_NAME_HOST_FILTER, DAG_NAME_ALIGNMENT, etc (also in this file)?

Reply:
Fixed, thanks.